### PR TITLE
SAK-46776 Ignite add ttl for cached objects

### PR DESCRIPTION
**Pull request by ern**
https://sakaiproject.atlassian.net/browse/SAK-46776

### CODE REVIEW

**Comment by bjones86** [comment]
Cool. Can you override the default via sakai.properties?

File: kernel/kernel-impl/src/main/webapp/WEB-INF/ignite-components.xml [hunk 1]
```diff
@@ -27,6 +29,21 @@
...
+    Expiry policy to for when cache objects are considered expired (ttl)
+    Default is 60 minutes from when object is created
+    -->
+    <bean id="org.sakaiproject.ignite.cache.expiryPolicy"
+          class="javax.cache.expiry.CreatedExpiryPolicy"
+          factory-method="factoryOf">
+        <constructor-arg>
+            <bean class="javax.cache.expiry.Duration">
+                <constructor-arg value="MINUTES"/>
+                <constructor-arg value="60"/>
```
Comment (ottenhoff) [inline kernel/kernel-impl/src/main/webapp/WEB-INF/ignite-components.xml:42]: I would propose a full day as caching time if right now is infinite.....

Comment (ern) [reply kernel/kernel-impl/src/main/webapp/WEB-INF/ignite-components.xml:42]: I like the idea of scoping data to the session expiration also I was wanting to come up with more selective approach to configuring individual caches as well. How's 120 mins?

Comment (ottenhoff) [reply kernel/kernel-impl/src/main/webapp/WEB-INF/ignite-components.xml:42]: Sure 120 mins works for me

**Comment by ern** [comment]
> Cool. Can you override the default via sakai.properties?

that will be another PR but yes you will be able to

**Pull request merged**